src/Content/Backend/Solution/Monaco.Template.Backend.Common.Application/Commands/Behaviors/CommandValidationBehavior.cs

```diff
@@ -113,15 +114,15 @@ public CommandValidationExistsBehavior(IValidator<TCommand> validator)
 		Validator = validator;
 	}
 
-	public virtual async Task<ICommandResult<TResult?>> Handle(TCommand request,
-															   RequestHandlerDelegate<ICommandResult<TResult?>> next,
-															   CancellationToken cancellationToken)
+	public virtual async Task<CommandResult<TResult?>> Handle(TCommand request,
+															  RequestHandlerDelegate<CommandResult<TResult?>> next,
+															  CancellationToken cancellationToken)
 	{
 		var validationResult = await Validator.ValidateAsync(request,
 															 options => options.IncludeRuleSets(ValidatorsExtensions.ExistsRulesetName),
 															 cancellationToken);
 		if (!validationResult.IsValid)
-			return new CommandResult<TResult?>(true, default);
+			return CommandResult<TResult?>.NotFound();
```

Comment:
Should this be ValidationFailed method rather than not found?

Reply:
No, because this is the `CommandvalidationExistsBehavior`, whose purpose is to validate if the element exists or not and if it doesn't, return the `CommandResult` with the `NotFound` flag set to true, which is what the old code was doing, hence the new code needs to call the `NotFound` method that returns the same value.

Reply:
All right then